src/UI/Internal/Dialog.elm

```diff
@@ -68,6 +67,12 @@ desktopDialogView cfg (Dialog.Dialog { title, icon } { body, buttons }) =
         [ body
         , buttonsRow cfg buttons
         ]
+        |> Element.el
+            [ Element.centerX
+            , htmlAttribute (Html.Attributes.style "position" "fixed")
+            , htmlAttribute (Html.Attributes.style "top" "50%")
+            , htmlAttribute (Html.Attributes.style "bottom" "50%")
```

Comment:
Does it not work when put in the `Element.column` attributes?

Reply:
No, has different behaviour